RELEASING.md

```diff
@@ -85,3 +84,24 @@ gpg --print-md SHA512 RELEASE_NOTES.html > RELEASE_NOTES.html.sha512
 2. Delete the staged distribution files from https://dist.apache.org/repos/dist/dev/storm/
 
 3. Delete the git tag.
+
+# How to vote on a release candidate
```

Comment:
I'm wondering if this belongs at the top of the file? The audience for it is probably broader than the audience for how to author a release.

Reply:
make sense